components/device-mgt/org.wso2.carbon.device.mgt.core/src/main/java/org/wso2/carbon/device/mgt/core/DeviceManagementPluginRepository.java

```diff
@@ -52,7 +52,7 @@ public DeviceManagementPluginRepository() {
     }
 
     public void addDeviceManagementProvider(DeviceManagementService provider) throws DeviceManagementException {
-        String deviceType = provider.getType();
+        String deviceType = provider.getType().toLowerCase();
```

Comment:
Just a suggestion, can't we include the toLowerCase() method to the getter/setter of the 'type' (ie, setType() /getType()) it self. If we do that, the we don't need to worry about performing this in the all the places.  Additionally, I feel it's better to handle this in the setter of 'type' (i.e, setType()) method as it's not used frequently as getter, and one time operation.